web/static.go

```diff
@@ -78,6 +78,21 @@ func root(c *Context, w http.ResponseWriter, r *http.Request) {
 func staticFilesHandler(handler http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("Cache-Control", "max-age=31556926, public")
+
+		if strings.HasSuffix(r.URL.Path, "/") {
```

Comment:
I take it, this check is not directly relevant?

Reply:
The diff makes it seem like I added something here, but really it's just duplicating the existing functionality for the new method. I'm assuming this was an attempt to prevent a directory listing or some such, but hadn't digged deeper beyond duplicating it.